Add report zoom to What's New in 9.3 (#59345)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v9-3.md

| 2022-10-20 12:00:00 | EU     | 2936  |
| 2022-10-20 01:00:00 | EU     | 912   |

## Reporting: Zoom in and out to fit your data better into a PDF

> This feature is availabe in Grafana Enterprise, Cloud Pro, and Cloud Advanced.

Reviewer: ```suggestion
This feature is available in Grafana Enterprise, Cloud Pro, and Cloud Advanced.
```

> Since dashboards appear on a screen and Reports are PDFs, it can be tricky to render data just the way you want to - maybe your report doesn't show enough columns in a table, or the titles appear too small. Now you can adjust the scale of your report, to zoom in and make each text field and panel larger or zoom out to show more data. 

Reviewer: ```suggestion
Because dashboards appear on a screen and reports are PDFs, it can be challenging to render data just the way you want to. Sometimes the report doesn't show enough columns in a table, or the titles appear too small. Now you can adjust the scale of your report to zoom in and make each text field and panel larger or zoom out to show more data. 
```

> Find Zoom in the "Format Report" section of your reporting configuration, and learn more about Reporting in our [docs]({{< relref "../enterprise/reporting/">}}).

Reviewer: ```suggestion
The zoom feature is located in the **Format Report** section of your reporting configuration. To learn more about reporting, refer to [Create and manage reports]({{< relref "../dashboards/create-reports/">}}).
```
